src/python/pants/backend/project_info/rules/fast_dependencies.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
We want to call this file `dependencies.py` for less of a code change when we finish porting V1 to V2.